api/v1/configurationpolicy_types.go

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2021 Red Hat, Inc.
 // Copyright Contributors to the Open Cluster Management project
 
+//nolint:tagliatelle
```

Comment:
Could you remove this and modify the linter exclusion rule to ignore `object-templates` instead? This will allow us to catch any future additions to the CRDs that don't conform.

```yaml
    - path: ^api/
      linters:
        - tagliatelle
      source: json:"policy-templates|json:"extra_vars
```

to:

```yaml
    - path: ^api/
      linters:
        - tagliatelle
      source: json:"object-templates
```

Reply:
Good idea